src/main/resources/dbmigration/eureka-ontology-EK_DEMOG-changelog.xml

```diff
@@ -2084,20 +2084,47 @@
             <column name="C_SYMBOL" value="DEM|SEX:Unspecified"/>
             <column name="EK_UNIQUE_ID" value="DEM|SEX:Unspecified"/>
         </insert>
-        <insert tableName="EK_DEMOG">
+        <insert tableName="EK_DEMOG" dbms="h2, oracle">
             <column name="C_HLEVEL" valueNumeric="2"/>
             <column name="C_FULLNAME" value="\EK\i2b2INTERNAL|Demographics\i2b2INTERNAL|Demographics|Age\"/>
             <column name="C_NAME" value="Age"/>
             <column name="C_SYNONYM_CD" value="N"/>
             <column name="C_VISUALATTRIBUTES" value="FAE"/>
-            <column name="C_BASECODE" value="i2b2INTERNAL|Demographics|Age"/>
+            <column name="C_BASECODE" value="DEM|Age"/>
             <column name="C_METADATAXML"/>
-            <column name="C_FACTTABLECOLUMN" value="concept_cd"/>
-            <column name="C_TABLENAME" value="concept_dimension"/>
-            <column name="C_COLUMNNAME" value="concept_path"/>
-            <column name="C_COLUMNDATATYPE" value="T"/>
-            <column name="C_OPERATOR" value="LIKE"/>
-            <column name="C_DIMCODE" value="\Cardiovascular Registry\i2b2INTERNAL|Demographics\i2b2INTERNAL|Demographics|Age\"/>
+            <column name="C_FACTTABLECOLUMN" value="patient_num"/>
+            <column name="C_TABLENAME" value="patient_dimension"/>
+            <column name="C_COLUMNNAME" value="birth_date"/>
+            <column name="C_COLUMNDATATYPE" value="N"/>
+            <column name="C_OPERATOR" value="&gt;"/>
+            <column name="C_DIMCODE" value="sysdate - (365.25 * 104)"/>
```

Comment:
Actually 105-120 is the highest range (they're out of order). Still, we probably need the following:

An >= 90 folder
A liquibase parameter that toggles whether to make the age ranges actual or HIPAA compliant. If HIPAA compliant, we want
85-89
90 and over
Omit the 85-94, 95-104, and 105-120 sub trees.

If actual, as-is except add a 90 and over folder.

Make the default "actual".

You'll need to alter provisioning to allow setting the new parameter.

Andrew

Sent from my iPhone

> On Aug 29, 2016, at 2:44 PM, Akshatha Pai <notifications@github.com> wrote:
> 
> In src/main/resources/dbmigration/eureka-ontology-EK_DEMOG-changelog.xml:
> 
> >              <column name="C_METADATAXML"/>
> > -            <column name="C_FACTTABLECOLUMN" value="concept_cd"/>
> > -            <column name="C_TABLENAME" value="concept_dimension"/>
> > -            <column name="C_COLUMNNAME" value="concept_path"/>
> > -            <column name="C_COLUMNDATATYPE" value="T"/>
> > -            <column name="C_OPERATOR" value="LIKE"/>
> > -            <column name="C_DIMCODE" value="\Cardiovascular Registry\i2b2INTERNAL|Demographics\i2b2INTERNAL|Demographics|Age\"/>
> > +            <column name="C_FACTTABLECOLUMN" value="patient_num"/>
> > +            <column name="C_TABLENAME" value="patient_dimension"/>
> > +            <column name="C_COLUMNNAME" value="birth_date"/>
> > +            <column name="C_COLUMNDATATYPE" value="N"/>
> > +            <column name="C_OPERATOR" value="&gt;"/>
> > +            <column name="C_DIMCODE" value="sysdate - (365.25 * 104)"/>
> 104 yrs because the ontology shows 95-104 to be the highest age range. attached the screenshot of the ontology. Does this have to be >=95 then?
> 
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>